### PR TITLE
:fuelpump: feat: add networking skills to roadmap

### PR DESCRIPTION
**Pull request by alizmhdi**
add networking skills to roadmap

### CODE REVIEW

**Comment by mostafaghadimi** [comment]
Hi @alizademhdi jan, thanks for your contribution. That’s LGTM. I wonder if it can be placed under the general software engineering skills or it should be considered as a separate topic. What’s your opinion?

**Comment by alizmhdi** [comment]
Hey Mostafa.
In general, I think networking should be classified in a different class than general software engineering skills. Because the types of problems that are currently in the general skills section of software engineering are a little different from networking.

**Comment by mostafaghadimi** [comment]
Approved. ✅

In case of agreement, please like this message.

Thanks
@ShAlireza @iamdorsa @niyushabaghayi @rezaaminimajd

**Comment by ShAlireza** [comment]
I'm not sure if knowing about VPNs and firewalls is a crucial skill a data engineer should have.
BTW, I totally agree with a general knowledge of famous protocols like HTTP(S) and IP stack.

**Comment by mostafaghadimi** [comment]
@ShAlireza one thing to mention is that many organizations use MFA (Multi-Factor Authentication) to keep their infrastructure secure. If we consider the data engineers as data infrastructure providers, I think it seems reasonable for them having idea about general concepts of security like firewall. In terms of AWS, we have security groups that Data/DevOps engineers should be familiar with.

**Comment by ShAlireza** [comment]
Agree!

File: roadmap.yaml [hunk 1]
```diff
@@ -9,6 +9,14 @@ Data Engineering:
...
     - Windows
     - Other UNIX distributions
+  Networking:
+    - Protocols: |
+        HTTP/HTTPS
+        TCP/IP
+        SSH
+        DNS
+    - VPN
+    - Firewalls
```
Comment (mostafaghadimi) [inline roadmap.yaml]: The value of `networking` should be key-value based (like dictionary) or list (valid yaml format). 

@alizademhdi Please change this one if it is possible for you. One suggestion could be:

```bash
Networking:
  Protocols:
    - HTTP/HTTPS
    - TCP/IP
    - SSH
    - DNS
  Concepts:    
    - VPN
    - Firewalls
```

**Review by mostafaghadimi — CHANGES_REQUESTED** [review]

**Comment by niyushabaghayi** [comment]
I think adding `VPN` and `Firewalls` aren't necessary for being a **Data Engineer**. These names are so general and being deep to them is for other roles like SRE, Infraustructure Engineers, etc.
BTW, I agree to add a `network` topic with a `protocols` sub-title.

@alizademhdi

File: roadmap.yaml (hunk 1 above)
Comment (alizmhdi) [reply roadmap.yaml]: fixed :)

**Review by mostafaghadimi — APPROVED** [review]
Well done!

**Pull request merged**